src/shortestpaths/dijkstra.jl

```diff
@@ -47,6 +47,7 @@ function dijkstra_shortest_paths(
     dists[srcs] = zero(T)
     pathcounts[srcs] = 1
     closest_vertices = Vector{U}()  # Maintains vertices in order of distances from source
+    popped_out = Vector{Bool}(zeros(nvg))   # popped_out[u]=true if shortest distance from src to u is found
```

Comment:
Well, in a PriorityQueue all the keys are stored twice once in the K,V pairs array and once in the Dict that has the indices. The vector of bool only stores 1 byte per entry so that will be more memory efficient but does